nbs/examples/Multivariate_with_TSMixer.ipynb

```diff
@@ -16,11 +16,11 @@
    "source": [
```

Comment:
<p>I only see the forecast for the MLPMultivariate in the plot, but the code seems to be generating the forecast for all models. Can you double check the plot shows the forecasts for all models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/938/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Should be fixed now</p>